src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -142,20 +144,20 @@ public override bool RunTask ()
 			return !Log.HasLoggedErrors;
 		}
 
-		static internal NativeAssemblerTargetProvider GetAssemblyTargetProvider (string abi)
+		static internal AndroidTargetArch GetAndroidTargetArchForAbi (string abi)
```

Comment:
This feels like an "odd" type to contain this method.  Wouldn't `NdkTools` make more sense?

Reply:
No, because it's not an NDK target arch, but the arch as used by the Android OS runtime. NDK uses different names for ARM targets, and the forms used here don't exist in NDK at all.